controllers/cre/controller.go

```diff
@@ -109,8 +109,8 @@ func (r *CustomRuntimeEnvironmentReconciler) reconcilePipelineRun(ctx context.Co
 	pipeline := build_types[cre.Spec.BuildType]
 	pipelineRun := &pipelinev1beta1.PipelineRun{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      fmt.Sprintf("cre-%s-%s", cre.GetName(), pipeline),
-			Namespace: cre.Namespace,
+			GenerateName: fmt.Sprintf("cre-%s-%s-", cre.GetName(), pipeline),
+			Namespace:    cre.Namespace,
 			Labels: map[string]string{
 				"cre.thoth-station.ninja/pipeline": build_types[cre.Spec.BuildType],
```

Comment:
@KPostOffice could you add a label refering to the CRE generation that created the PipelineRun